enhancements/declarative-index-config.md

```diff
@@ -173,25 +178,26 @@ $ cat etcd.json
     "version": "0.9.2",
     "properties":[
         {
-            "name": "olm.package",
-            "values": {
+            "type": "olm.package.provided",
```

Comment:
Catalog operator infers `olm.package` based on the first-class fields if it doesn't find an `olm.package` property, so I don't think this rename would cause problems.

Reply:
https://github.com/operator-framework/enhancements/pull/69/files#r592511531